Anime-App/Feature/Anime/View/ViewControllers/AnimeViewController.swift

```diff
@@ -102,12 +114,35 @@ extension AnimeViewController: UISearchBarDelegate {
 
 extension AnimeViewController {
     
+    func addHapticFeedBack() {
+        let hapticFeedbackIsOn = UserDefaults.standard.bool(forKey: "UseHaptic")
+        if hapticFeedbackIsOn {
+            print("haptic feedback is on")
+            let generator = UIImpactFeedbackGenerator(style: .heavy)
+            generator.impactOccurred()
+        }
+    }
+    
+    @objc func addSound() {
+        let audioIsOn = UserDefaults.standard.bool(forKey: "UseAudio")
+        addHapticFeedBack()
+        if audioIsOn {
+            print("Sound is playing")
+            audioManager.playSound()
+        } else {
```

Comment:
So you don't actually need this else block you want to just copy what you did with the haptic feedback where if it's on just play the sound